easybuild/easyconfigs/r/rpy2/rpy2-2.4.4-goolf-1.4.10-Python-2.7.5.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = "PythonPackage"
+
+name = 'rpy2'
+version = '2.4.4'
+
+homepage = 'http://rpy.sourceforge.net/'
+description = """rpy2 is a redesign and rewrite of rpy. It is providing a low-level interface 
+ to R from Python, a proposed high-level interface, including wrappers to graphical libraries, 
+ as well as R-like structures and functions. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://pypi.python.org/packages/source/r/rpy2/']
+sources = [SOURCE_TAR_GZ]
+
+python = 'Python'
+pyver = '2.7.5'
+versionsuffix = '-%s-%s' % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+    ('R', '3.1.0')
```

Comment:
Jenkins says no on current state:

```
R/3.1.0-goolf-1.4.10
```

We do have `3.0.2` with that toolchain. And we have `3.1.2` easyconfigs that come close (goolf*)